.travis.yml

```
@@ -9,8 +10,7 @@ notifications:
   webhooks:
     - https://webhooks.gitter.im/e/b5d48907cdc89864b874
 rvm:
-  - 2.5.0
-  - 2.4.1
-  - 2.3.3
-  - 2.2.9
+  - 2.6.1
+  - 2.5.3
+  - 2.4.5
```

What if we don't specify bugfix version, does it runs against latest minor version?

```suggestion
  - 2.6
  - 2.5
  - 2.4
```

---

Running the build now.